packages/imitation_game/README.md

```diff
@@ -93,11 +93,14 @@ An implementation has to follow these rules:
 A single test run can report multiple numbers.
 
 ## Results
-Date created: 2020-08-17 23:57:16.702500Z
+Date created: 2020-08-19 20:46:17.734351Z
 
 - smiley
     - flutter
-      - startupTime: 0.561475s
+      - startupTime: 0.691717
```

Comment:
As mentioned in [another comment][1], we may need to add an OS subdirectory as `smiley/android/flutter` and `smiley/ios/flutter`. Otherwise, it will be confusing if the `startupTime` is from iOS or Android as they're different. For example, the `startupTime` here is from iOS while the `adb_memory_total` is from Android. However, it's very easy to think that `startupTime` here is also coming from Android.

[1]: https://github.com/flutter/packages/pull/181#discussion_r460359393

Reply:
Yea, I thought about that.  I think we can just disambiguate the measurements with names like "ios_startup_time".  I changed that measurement's name.